plugins/uefi-capsule/fu-uefi-capsule-plugin.c

```diff
@@ -619,6 +619,24 @@ static FuVolume *
 fu_uefi_capsule_plugin_get_default_esp(FuPlugin *plugin, GError **error)
 {
 	g_autoptr(GPtrArray) esp_volumes = NULL;
+	const gchar *recovery_partitions[] = {
```

Comment:
leave a comment where the list came from near this so we can sync it later if a new one is missing.

Reply:
Done.